src/Expression.php

```diff
@@ -648,14 +648,18 @@ public function getIterator()
      *
      * @return array
      */
-    public function get()
+    public function get(string $classname = 'stdClass', $returnObject = false)
```

Comment:
Seems like a bad design - why two parameters if one is enought? And you should use `\stdClass::class` instead of string.